app/index.js

```diff
@@ -61,11 +61,42 @@ module.exports = Generator.extend({
     })
   },
 
+  _isTrue: function (value) {
+    return (value === true || value === 'true')
+  },
```

Comment:
make this a local function, we only use it in `initSpec`

Reply:
done